std/experimental/ndslice/slice.d

```diff
@@ -1181,7 +1188,7 @@ struct Slice(size_t _N, _Range)
     }
 
     static if (!hasPtrBehavior!PureRange)
-    this(in size_t[PureN] lengths, in sizediff_t[PureN] strides, PtrShell!PureRange shell)
+    this(in size_t[PureN] lengths, in sizediff_t[PureN] strides, PtrShell!PureRange shell) @system
```

Comment:
I'm not 100% sure but I *think* that as members of a template struct, `@system` should be inferred here. Regardless, it's the default anyway.